index.html

```diff
@@ -0,0 +1,39 @@
+<html>
+
+<head>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+	<link rel="stylesheet" href="index.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Karla:wght@400;800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+	<header>
+		<div class="header-inner">
+			<img src="./images/pumpkin.png" class="pumpkin-img">
+			<div>
+				<h2>Pumpkin's Purrfect...</h2>
+				<h1>Meme Picker</h1>
+			</div>
+		</div>
+	</header>
+	<main>
+		<h3>Select Your Current Emotion</h3>
+		<div class="controls-container">
```

Comment:
I would make this a form:

```suggestion
		<form class="controls-container">
```

and attach the event `renderCat` function to the form's `submit` event. You just need to make sure to `event.preventDefault()` in the `renderCat` function so the page doesn't redirect

Reply:
I can't seem to get the preventDefault working, might need some help with this